perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -33,6 +34,8 @@
 flags.DEFINE_string('fio_jobfile', 'fio.job', 'job file that fio will use')
 flags.DEFINE_integer('memory_multiple', 10,
                      'size of fio scratch file compared to main memory size.')
+flags.DEFINE_boolean('against_device', False,
```

Comment:
Thinking more about this, we will likely need to add another flag here:

flags.DEFINE_integer(device_fill_size, -1, "the amount of bytes to write to the device when against_device is set to true. Default -1 means fill the entire device.")

This way, for some providers that do not require pre-warm of the volume, the user can use this parameter to only fill part of the device, e.g., 10GB, and then use that part for subsequent read testing (by specifying --size=10GB in their read job).